docs/spec/openapi.yml

```diff
@@ -1635,6 +1674,38 @@ components:
         password:
           type: string
           example: root
+    DatabaseClusterComponents:
+      type: array
+      description: components related data
+      items:
+        type: object
+        x-go-type-name: DatabaseClusterComponent
+        properties:
+          status:
+            type: string
+          name:
+            type: string
+          type:
+            type: string
+          age:
+            type: string
+          restarts:
+            type: integer
+          ready:
+            type: string
+          containers:
+            type: array
+            items:
+              $ref: '#/components/schemas/DatabaseClusterComponentContainer'
+    DatabaseClusterComponentContainer:
+      type: object
+      properties:
+        name:
+          type: string
+        age:
+          type: string
```

Comment:
Maybe we should use a CreationTimestamp instead of the age, this way the frontend can continuously update the age between requests.

Reply:
Updated